kuma/wiki/content.py

```diff
@@ -493,7 +493,7 @@ def __iter__(self):
             if token['type'] == 'StartTag' and token['name'] == 'a':
                 attrs = dict(token['data'])
                 names = [name for (namespace, name) in attrs.keys()]
-                for (namespace, name), value in attrs.items():
+                for (namespace, name), value in attrs.copy().items():
```

Comment:
Is ``copy()`` needed here?

Reply:
Yes because the original `attrs` is changed during the iteration which is no longer allowed in Python 3.